support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -82,7 +82,7 @@ export const tests: Tests = {
 		seed: 2,
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 	},
-	additionalBenefits: {
+	additionalOffers: {
```

Comment:
Should this be more specific? e.g. `usFreeBookOffer`? Just wondering about any tracking we might be doing. 

non-blocking.

Reply:
Can see `offers` possibly being used in the future so perhaps `offersUSBook` would be ok?

Reply:
Yes thats more descriptive, renamed